src/services/facilityService.ts

```diff
@@ -18,14 +19,31 @@ export const getFacilityById = async (id: string) : Promise<Facility | null> =>
     return convertedEntity
 }
 
-export const addFacility = async (facility : Facility) : Promise<void> => {
-    //todo
+export const addFacility = async (input: FacilityInput) : Promise<Facility> => {
+    const db = getFirestore()
+   
+    const facilityRef = db.collection('facilities')
+
+    const healthcareProfessionalIds = 
+    await mapAndValidateHealthcareProInput(input.healthcareProfessionals as HealthcareProfessionalInput[])
```

Comment:
nit: should stay consistent with either professsional or pro

Reply:
The reason for the change was because eslint was complaining that it was too long 😅